vault/resource_kmip_secret_role.go

```diff
@@ -0,0 +1,287 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var kmipRoleAPIBooleanFields = []string{
+	"operation_activate",
+	"operation_add_attribute",
+	"operation_all",
+	"operation_create",
+	"operation_destroy",
+	"operation_discover_versions",
+	"operation_get",
+	"operation_get_attribute_list",
+	"operation_get_attributes",
+	"operation_locate",
+	"operation_none",
+	"operation_register",
+	"operation_rekey",
+	"operation_revoke",
+}
+
+func kmipSecretRoleResource() *schema.Resource {
+	return &schema.Resource{
+		Create: kmipSecretRoleCreate,
+		Read:   kmipSecretRoleRead,
+		Update: kmipSecretRoleUpdate,
+		Delete: kmipSecretRoleDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"path": {
```

Comment:
Does Vault support renaming the path? Should this be `ForceNew` as well?

Reply:
Resolving since this has been captured in remounting work 52cc467